### DISCUSSION TITLE
AAD Connect - .local -> veryfied domain po userProfileName

### DISCUSSION THREAD

**Discussion by phaate** [AzureCommunityPL/group#170] [Pomoc]
Cześć!

Chcę zrobić sync'a z użyciem AAD Connect z domeny AD .local.
Wiadomo, że bez dodania sufiksu na zweryfikowaną domenę utworzy mi konta z sufiksem onmicrosoft.com. 

Chciałbym jednak żeby przy użyciu userProfileName poszedł mi sync do AAD ale konto utworzone po AAD było z sufiksem zweryfikowanej domeny @global.com a nie @tenant.onmicrosoft.com. Ma ktoś pomysł jak to ogarnąć?

edit. Konta lokalne **muszą zostać z sufiksem .local**

**Comment by eklime** [comment]
https://docs.microsoft.com/en-us/microsoft-365/enterprise/prepare-a-non-routable-domain-for-directory-synchronization

**Reply by phaate** [reply]
Właśnie rzecz w tym, że konta w AD mają zostać .local a konta w AAD mają być w domenie @global.com.

No i parametrem do synca ma być uPN a nie mail. Z mailem sprawa wydaje się być prosta.
